javascript/documentation/library-customization.rst

```diff
@@ -206,11 +208,15 @@ Framework models
 ~~~~~~~~~~~~~~~~
 
 The ``semmle.javascript.frameworks.HTTP`` module defines many abstract classes that can be extended
-to implement support for new web frameworks. These classes, in turn, are used by some of the
-security queries (such as the cross-site scripting queries) to define sources and sinks, so these
-queries will automatically benefit from the additional modeling.
+to implement support for new web server frameworks. These classes, in turn, are used by some of the
+security queries (such as the reflected cross-site scripting query) to define sources and sinks, so
+these queries will automatically benefit from the additional modeling.
+
+Similarly, the ``semmle.javascript.frameworks.ClientRequests`` module defines an abstract class for
+modeling client-side HTTP requests. It comes with built-in support for a number of popular
+frameworks, and you can add support for new frameworks by extending the abstract class.
```

Comment:
Perhaps mention the class that should be extended:  "by extending the abstract class `CustomClientRequest`."

Reply:
I didn't put in the name because I'd like to rename it to `ClientRequest::Range` first (keeping the old name as a deprecated alias).